Dockerfile

```diff
@@ -12,10 +12,39 @@ RUN npm install
 
 COPY . .
 
-FROM nginx:1.19.3
+RUN npm run build:prod
 
-COPY --from=builder /usr/src/app/dist/web-app /usr/share/nginx/html
+RUN ls -la /usr/src/app/dist/web-app
 
-EXPOSE 80
+# Stage 2: Host Web App on OpenLiteSpeed
+FROM litespeedtech/openlitespeed:latest AS runner
```

Comment:
Can we  have a separate Dockerfile using OpenLiteSpeed server while keeping Nginx as default?